doc/linux/build_instructions.md

```diff
@@ -77,3 +77,32 @@ make
 ## SDK with bindings
 
 - Please check the readme files for each type of binding in the [bindings directory](../../bindings).
+
+### Enabling the point cloud display in aditof-demo
+
+ OpenCV needs to be build manually in order to visualize pointCloud data because the prebuild libraries do not come with the viz module installed.
+ The steps required for building and including the required dependencies are presented below:
+ 
+ Requirements:
+ * OpenCV 4.5.0
+
+```console
+sudo apt-get install libgtk-3-dev libvtk6-dev
+git clone --branch 4.5.0 --depth 1 https://github.com/opencv/opencv.git
+git clone --branch 4.5.0 --depth 1 https://github.com/opencv/opencv_contrib.git
+cp -r .\opencv_contrib\modules\viz .\opencv\modules\
```

Comment:
backslash problem in my case:
cp: cannot stat '.opencv_contribmodulesviz': No such file or directory
cp: cannot stat '.opencvmodulescd': No such file or directory

Reply:
Fixed, thank you!